Jenkinsfile

```diff
@@ -1,21 +1,25 @@
 node('ios-osx') {
     stage("Checkout") {
         checkout scm
+
         withCredentials([file(credentialsId: 'ceb2d5e9-fc88-418f-aa65-ce0e0d2a7ea1', variable: 'CLIQZ_CI_SSH_KEY')]) {
             sh '''#!/bin/bash -l -x
-            mkdir -p ~/.ssh
-            cp $CLIQZ_CI_SSH_KEY ~/.ssh/id_rsa
-            chmod 600 ~/.ssh/id_rsa
-            echo $CLIQZ_CI_SSH_KEY
-            ssh-keyscan -t rsa github.com >> ~/.ssh/known_hosts
-            git submodule update --init
+                set -e
+                set -x
+                mkdir -p ~/.ssh
+                cp $CLIQZ_CI_SSH_KEY ~/.ssh/id_rsa
+                chmod 600 ~/.ssh/id_rsa
+                echo $CLIQZ_CI_SSH_KEY
+                ssh-keyscan -t rsa github.com >> ~/.ssh/known_hosts
+                git submodule update --init
             '''
         }
     }
 
     stage('Prepare') {
-
-        sh '''#!/bin/bash -l -x
+        sh '''#!/bin/bash -l
+            set -e
```

Comment:
From Jenkins log it looks like we shouldn't set this flag for this step.

Reply:
why exactly? please point to exact log line or snippet

Reply:
Sorry... The reason is this line current stops the build:
```+ brew install xctool

Error: xctool 0.3.1 is already installed

To upgrade to 0.3.2, run `brew upgrade xctool`

script returned exit code 1```
[source](http://jenkins-master-v2-gp.clyqz.com:8080/blue/organizations/jenkins/cliqz-oss%2Fbrowser-ios%2Fbrowser_ios_test/detail/PR-305/2/pipeline)